mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -185,7 +185,7 @@ APCGun.AA:
 	Report: gun20.aud
 	ValidTargets: Air
 	Projectile: Bullet
-		Speed: 1c682
+		Speed: 2
```

Comment:
Going from the changelog, I assume you wanted `2c0` here, because that would be an increase.